mlir-dialect-experiments/exec-mlir-from-rust/build.rs

```diff
@@ -0,0 +1,6 @@
+use std::process::Command;
+
```

Comment:
Apart from the make style issues, the Rust build script should not invoke make. The CMake part should be responsible for building the library, when it exists corrosionm should invoke cargo which then can assume that the library exists.